tsconfig.json

```diff
@@ -6,8 +6,13 @@
     "declaration": true,
     "moduleResolution": "node",
     "experimentalDecorators": true,
-    "resolveJsonModule": true
+    "resolveJsonModule": true,
+    "lib": [
+      "dom",
+      "dom.iterable",
```

Comment:
dom ?

Reply:
not sure, but i thought it was a good idea to use the same libraries as the integration-tests 🤷